web-frontend/src/utils/MassaFormating.tsx

```diff
@@ -67,3 +71,74 @@ export function maskAddress(str: string, length = 4, mask = '. . .'): string {
 
   return str?.substring(0, start) + mask + str?.substring(end);
 }
+
+/**
+ * Validates a recipient address.
+ * @param recipient - The recipient address to validate.
+ * @param amount - The amount to validate.
+ * @param balance - The balance to validate.
+ * @returns An object containing the error message if the address is invalid, `null` otherwise.
+ */
+
+export function validate(
```

Comment:
the tsdoc, the name of the function are not in accordance to what the function really do